frontend/src/components/comparison/comparison-data/ComparisonData.css

```diff
@@ -37,7 +37,8 @@
 /* BASIC INFO */
 .comparison-product-image {
 	display: block;
-	width: 100%;
```

Comment:
The comparison page images still have varying heights? Am I looking at the wrong thing?

<img width="1198" alt="Screenshot 2024-03-19 at 12 19 01 AM" src="https://github.com/tzada8/capstone/assets/57775285/1ab660c0-e491-483b-889f-729d3dcc966b">

Reply:
I'm just going to explicitly set the height to `300px` for now. When I do QA I'll see if this doesn't work for some cases